test/unit/testhelpers.ts

```diff
@@ -43,7 +43,7 @@ export const setupFakeApiError = (
 export const setupFakeInvalidStateError = (): AxiosError => {
     let err = setupFakeApiError(409,
         'Job is in invalid state',
-        'https://www/rev.ai/api/speech/v1/errors/invalid-job-state',
+        'https://www.example.com/api/speech/v1/errors/invalid-job-state',
```

Comment:
Instead of changing these to `example.com` to satisfy link checker ignores - should fix to point to the actual error code links - which is at `https://www.rev.ai/api/v1/errors/<error-code>` so here it would be
```suggestion
        'https://www.rev.ai/api/v1/errors/invalid-job-state',
```

Please fix for the others

Reply:
these redirect to `https://docs.rev.ai/api/asynchronous/#error-codes` is that meant to happen or should these links be to a page that shows this specific error and one you can see if you check the console/network when inspecting?

Reply:
It's expected - that's where all of our error codes are/will be documented